config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
   resources :posts
   devise_for :users
+  
   root 'pages#index'
-  get '/pages' , to: 'pages#index'
-  get 'pages/meetup' , to: 'pages#meetup'
-  get 'pages/contact' , to: 'pages#contact'
-  get 'pages/about' , to: 'pages#about'
+  get '/meetup' , to: 'pages#meetup'
```

Comment:
Nice job cleaning these up, I didn't realize that was an option, I'll for sure use this approach onwards.

Reply:
Yeah, I like keeping things nice a tidy in the address bar if possible. There is no other effect code wise.